src/applications/third-party-app-directory/containers/ThirdPartyAppList/index.jsx

```diff
@@ -45,11 +43,10 @@ export class ThirdPartyAppList extends Component {
     // Show the error alert box if there was an error.
     if (error) {
       return (
-        <AlertBox
-          headline="Something went wrong"
-          content={error}
-          status="error"
-        />
+        <VaAlert visible status="error">
```

Comment:
Wouldn't it be better to just use the `va-alert` version of the web-component here? There isn't a need to use the react bindings for this

Reply:
I think you can just use `va-alert` for all of these since none of them have events attached to them